packages/middleware-flexible-checksums/src/configuration.ts

```diff
@@ -31,6 +32,16 @@ export interface PreviouslyResolved {
    */
   md5: ChecksumConstructor | HashConstructor;
 
+  /**
+   * Determines when a checksum will be calculated for request payloads
+   */
+  requestChecksumCalculation: Provider<string>;
+
+  /**
+   * Determines when a checksum will be calculated for response payloads
+   */
+  responseChecksumValidation: Provider<string>;
```

Comment:
should this use the enum instead of string?

Reply:
Looks like it was missed in config resolver PR.
I added it in https://github.com/aws/aws-sdk-js-v3/pull/6489, so that it's out of this PR which processes those values.

Reply:
Fixed in https://github.com/aws/aws-sdk-js-v3/pull/6489, and rebased.